internal/pkg/aws/s3/s3.go

```diff
@@ -218,6 +221,15 @@ func (s *S3) upload(bucket, key string, buf io.Reader) (string, error) {
 	return resp.Location, nil
 }
 
+func defaultContentTypeFromExt(key string) *string {
+	contentType := mime.TypeByExtension(filepath.Ext(key))
+	if contentType == "" {
+		return nil
+	}
+	fmt.Println(contentType)
```

Comment:
:doge Just to be annoying and point out trivial things

Reply:
:homer-disappearing: